<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>file_browser_dialog</class>
 <widget class="QDialog" name="file_browser_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="browse_files_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>browse files</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="file_browser_text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>701</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="file_list">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>231</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_directories_button">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>browse directories</string>
   </property>
  </widget>
  <widget class="QListWidget" name="directory_list">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>231</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="filelist_list">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>210</y>
     <width>231</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_filelists_button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>170</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>browse lists</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
